week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,23 @@
 /** Write your CSS in here **/
+body {
+  background-color: aliceblue;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+}
+
+div {
+  margin-top: 100px;
+  border-radius: 20px;
+  padding: 20px;
+}
+
+h3 {
+  padding: 30px;
+}
+
+button {
+  background-color: aqua;
+  margin-top: 20px;
```

Comment:
styling looks a bit weird as the UX feels tiring trying to find the button after each click

Reply:
Modified style, fixed the next button's position.